src/components/DuosHeader.jsx

```diff
@@ -0,0 +1,540 @@
+
+  useEffect(() => {
+    setSelectedMenuTab(initialTab === -1 ? false : initialTab);
+    setSelectedSubTab(initialSubTab === -1 ? false : initialSubTab);
+  }, [initialTab, initialSubTab]);
+
+  return (
+    <div className={`navbar-logged ${orientation === 'vertical' ? 'navbar-vertical' : ''}`}>
+      {makeNotifications()}
+      <ul className="navbar-main">
+        <div style={{ width: '100%', display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
+          <Link
+            isRendered={orientation === 'horizontal'}
+            id="link_logo"
+            to="/home"
+            className="navbar-brand"
+          >
+            <img style={duosLogoImage} src={DuosLogo} alt="DUOS Logo" />
+          </Link>
+          {
+            isLogged && (
+              <Box className="duos-navigation-box">
+                <Tabs
+                  value={selectedMenuTab}
+                  variant="scrollable"
+                  scrollButtons="auto"
+                  orientation={orientation}
+                  TabIndicatorProps={{ style: { background: '#2BBD9B' } }}
+                >
+                  {tabs.map((tab, tabIndex) => (
+                    <Tab
+                      key={`${tab.link}_${tabIndex}`}
+                      label={tab.label}
+                      style={selectedMenuTab === tabIndex ? styles.mainTabActive : styles.mainTab}
+                      to={{ pathname: tab.link, state: { selectedMenuTab: tabIndex } }}
+                      component={Link}
+                    />
+                  ))}
+                </Tabs>
+              </Box>
+            )
+          }
+          {
+            !isLogged && (
+              <ul className="navbar-public">
+                <li>
+                  <a
+                    id="link_about"
+                    className="navbar-duos-link"
+                    href="https://broad-duos.zendesk.com/hc/en-us/articles/360060400311-About-DUOS"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    <div className="navbar-duos-icon-about" style={navbarDuosIcon}></div>
+                    <span style={navbarDuosText}>About</span>
+                  </a>
+                </li>
+                <li>
+                  <a
+                    id="link_help"
+                    className="navbar-duos-link"
+                    href="https://broad-duos.zendesk.com/hc/en-us/articles/360059957092-Frequently-Asked-Questions-FAQs-"
+                    target="_blank" rel="noreferrer"
+                  >
+                    <div className="navbar-duos-icon-help" style={navbarDuosIcon}></div>
+                    <span style={navbarDuosText}>FAQs</span>
+                  </a>
+                </li>
+                {contactUsButton}
+                {supportrequestModal}
+              </ul>
+            )
+          };
+        </div>
+        {isLogged && (
+          <div
```

Comment:
Can we add back any missing comments?
```suggestion
           // Navbar right side
           <div
```